BUILD.md

```diff
@@ -36,12 +36,28 @@ make dockerinit
 make dockerdev
 ```
 
+If you do not want to build the docker image locally, or are not working on it, it may be more straightforward to
+pull the image from docker instead of building it. This should be a bit faster and there should not be any issues with
+`make dockerdev` expecting specific version of the image.
+
+```sh
+docker pull shiftcrypto/firmware_v2:latest
```

Comment:
So sorry for missing this :innocent:, but since last week we don't use `:latest` but the version in `.containerversion`. See https://github.com/BitBoxSwiss/bitbox02-firmware/pull/1282/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R134

Please change this line to `make dockerpull`.

Reply:
Will amend the commit so it stays as 1 in a minute